stable/chartmuseum/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 2.3.2
+version: 2.3.3
```

Comment:
I have a consider on this, the pr changed the source apiVersion. So it just works for `Kubernetes 1.6`?  Can the changes back forward compatibility? If not, we may need to bump the major version and make add tips in README.md.

Reply:
From what I understand, this is backward compatible. The `extensions/v1beta1` API does not contain the _Deployment_ resource anymore in v1.16. Instead, it is available in `apps/v1` ever  since v1.9. Ingress will be removed from `extensions/v1beta1` in v1.20 and was available in `networking.k8s.io/v1beta1 ` ever since v1.14. Maybe bumping the **minor** would be more appropriate since it is still backward compatible?

```diff
- version: 2.3.2
+ version: 2.4.0
```

Reply:
Yes, thanks.